composer.json

```diff
@@ -10,15 +10,18 @@
         }
     ],
     "require": {
+        "acquia/acquia_cms": "^1.1",
         "acquia/blt": "^12",
         "acquia/blt-phpcs": "^1.0",
-        "acquia/lightning": "^5",
+        "acquia/drupal-environment-detector": "^1",
+        "acquia/memcache-settings": "^1",
```

Comment:
These came from the [Acquia CMS Project](https://github.com/acquia/acquia-cms-project/blob/develop/composer.json). I'm not sure why the project is pulling them in and not `acquia/acquia_cms` directly. @msherron might know?

Reply:
@balsama - those to packages have scaffolding changes they need to make in settings.php, and they did not work as expected when I only relied on [acquia/acquia_cms to make the changes](https://github.com/acquia/acquia_cms/blob/develop/composer.json#L22) (despite having acquia_cms in the allowed-packages array in acquia-cms-project). Maybe I missed something, but that was the workaround.